package.json

```diff
@@ -51,13 +53,12 @@
     "graphql": "^15.7.2",
     "graphql-iso-date": "^3.6.1",
     "graphql-parse-resolve-info": "^4.12.0",
-    "graphql-redis-subscriptions": "^2.4.0",
     "graphql-relay": "^0.9.0",
     "graphql-subscriptions": "^2.0.0",
     "graphql-tools": "^4.0.8",
     "he": "^1.2.0",
     "highlight.js": "^11.3.1",
-    "ioredis": "^4.28.2",
+    "ioredis": "^5.0.5",
```

Comment:
I think we don't need this package directly anymore. We use it indirectly through other packages, don't we?